exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/data/utils/flight_utils.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<p>And, additionally everything except the UI stuff should go into the notebook connector. And, about the UI stuff we talk separately, because I think, I now know how to extract it to the notebook connector and test it properly there.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/306/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>It makes no sense to me to move this piece of code to the notebook connector. This is not a component of the system. It's just a data pre-pseprocessing for a particular example. If we decide to use a different dataset for this demo there will be a different pre-processing code.</p>